charts/values.yaml

```diff
@@ -0,0 +1,238 @@
+    ##
+    size: 10Gi
+    ## @param primary.persistence.selector Selector to match an existing Persistent Volume (this value is evaluated as a template)
+    ## selector:
+    ##   matchLabels:
+    ##     app: my-app
+    ##
+    selector: {}
+  ## @section PostgreSQL Primary parameters
+  ##
+  primary:
+    ## PostgreSQL Primary service configuration
+    ##
+    service:
+      ## @param primary.service.ports.postgresql PostgreSQL service port
+      ##
+      ports:
+        postgresql: 5432
+
+
+## @section mysql Parameters
+##
+
+## mysql properties
+##
+mysql:
+  ## @param enabled Enable mysql
+  enabled: false
+  ## Properties for using an existing mysql installation
+  ## @param mysql.image.registry mysql image registry
+  ## @param mysql.image.repository mysql image repository
+  ## @param mysql.image.tag mysql image tag (immutable tags are recommended)
+  ## @param mysql.image.pullPolicy mysql image pull policy
+  ## @param mysql.image.pullSecrets Specify image pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/mysql
+    tag: 8.0.28-debian-10-r23
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## Example:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+  ## @param commonLabels [object] Common labels to add to all MySQL resources (sub-charts are not considered). Evaluated as a template
+  commonLabels:
+    app: clusterpedia-internalstorage
+    internalstorage.clusterpedia.io/type: mysql
+  auth:
+    ## @param auth.rootPassword Password for the `root` user. Ignored if existing secret is provided
+    ##
+    rootPassword: "dangerous"
+    ## @param auth.database Name for a custom database to create
+    ##
+    database: clusterpedia
+    ## @param auth.username Name for a custom user to create
+    ##
+    username: ""
+    ## @param auth.password Password for the new user. Ignored if existing secret is provided
+    ##
+    password: ""
+  ## Enable persistence using Persistent Volume Claims
+  ##
+  persistence:
+    ## @param primary.persistence.enabled Enable persistence on MySQL primary replicas using a `PersistentVolumeClaim`. If false, use emptyDir
+    ##
+    enabled: true
+    ## @param primary.persistence.size MySQL primary persistent volume size
+    ##
+    size: 10Gi
+    ## @param primary.persistence.selector [object] Selector to match an existing Persistent Volume
+    ## selector:
+    ##   matchLabels:
+    ##     app: my-app
+    ##
+    selector: {}
+  ## @section MySQL Primary parameters
+  primary:
+    ## MySQL Primary Service parameters
+    ##
+    service:
+      ## @param primary.service.port MySQL Primary K8s service port
+      ##
+      port: 3306
```

Comment:
If the values of the subcharts are the same as the default values can they be ignored in values.yaml to avoid too much complexity?